src/ugeneui/src/main_window/SplashScreen.cpp

```diff
@@ -86,7 +86,7 @@ SplashScreenWidget::SplashScreenWidget() {
     setObjectName("splash_screen_widget");
 
     Version v = Version::appVersion();
-    version = v.text;
+    version = QString::number(v.major) + (v.minor == 0 ? "" : "." + QString::number(v.minor)) + (v.suffix.isEmpty() ? "" : "-" + v.suffix);
```

Comment:
if v.major == 38 and v.minor == 0 then
   version will be equal 380-<suffix>

Doesn't it?

Reply:
When minor is 0 the suffix like ".0" should not be shown at all. 
This way we have "38" for the version 38.0 and "38.1" for the version 38.1

Reply:
if v.major == 38 and v.minor == 0 and the suffix is "dev" then the version text shown on splash screen will be "38-dev"

Reply:
O, yes!
I see now the `))`
You need to format the line
We have a rules!

Reply:
It's > 80 chars length

Reply:
$ echo 'version = QString::number(v.major) + (v.minor == 0 ? "" : "." + QString::number(v.minor)) + (v.suffix.isEmpty() ? "" : "-" + v.suffix);' | wc -c
136

Reply:
I reformatted this piece of code to be shorter.

We have clang-format that has all formatting rules. I try to apply it always to the files before submit.
It has no 80-chars rule and I'm not sure we really need it: it will make the code less readable because we will have to use either shortened unreadable variable names or a lot of forced wrapping in unexpected places.

If you care about code review experience with 1920px screen GitHub fits 120 characters per line in diffs with no wraps. This limit looks reasonable to me.

When, for some reason you have a smaller screen you can switch into "unified" diff mode - that will show even more characters per line.